src/JoomlaBrowser.php

```diff
@@ -192,6 +192,7 @@ public function installJoomla()
         //endif;
         $I->selectOption(['id' => 'jform_sample_file'], ['id' => 'jform_sample_file0']); // No sample data
         $I->click(['link' => 'Install']);
+        $I->wait(1);
```

Comment:
same here, we are waiting for text in line 199 so I think this one is already covered.